spec/cypress/integration/lead_providers/ReportSchools.feature

```diff
@@ -0,0 +1,4 @@
+Feature: Report Schools flow
+  Scenario: Visiting the start page
+    Given I am on "lead providers report schools start" page
+    Then the page should be accessible
```

Comment:
I think I'd prefer this to start on the dashboard, to prove the journey joins up. Is everything in place to do that?

Reply:
Well, not quite, but it wasn't too hard to set it up.